components/led_strip_spi_apa102/include/led_strip_spi_apa102.h

```diff
@@ -0,0 +1,36 @@
+#ifndef led_strip_spi_apa102_H
+#define led_strip_spi_apa102_H
+
+#include <stdio.h>
+#include <string.h>
+#include <math.h>
+#include <stdlib.h>
+
+// #include "freertos/FreeRTOS.h"
+// #include "freertos/task.h"
+// #include "esp_system.h"
+// #include "driver/gpio.h"

```

Comment:
please remove these header if you don't need it

Reply:
I removed the commented out headers, sorry.
Yeah I'm going to do the second stage bootloader as a seperate PR :)